src/ResourceManager/Maps/Maps.Test/Commands.Maps.Test.csproj

```diff
@@ -155,13 +159,29 @@
     <None Include="ScenarioTests\MapsAccountTests.ps1">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Microsoft.Azure.Commands.Maps.Test.ScenarioTests.MapsAccountTests\TestGetAccountKeys.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="SessionRecords\Microsoft.Azure.Commands.Maps.Test.ScenarioTests.MapsAccountTests\TestGetAccounts.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="SessionRecords\Microsoft.Azure.Commands.Maps.Test.ScenarioTests.MapsAccountTests\TestNewAccount.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
??

Reply:
Similar to MachineLearning, the Maps tests weren't being run during the build, but I was able to make a few changes to get them to run successfully on the build. This included:
- Adding their `.json` recordings to their test project
- Updating the ResourceManagementClient instantiated in their test controller
- Making sure the Resources project was built with the Maps solution